ssl/ssl_locl.h

```diff
@@ -551,6 +551,8 @@ struct ssl_session_st {
         /* Session lifetime hint in seconds */
         unsigned long tick_lifetime_hint;
         uint32_t tick_age_add;
+        unsigned char *tick_nonce;
+        size_t tick_nonce_len;
```

Comment:
It's consistent with how we've stored all the other things like this. For example compare with ecpointformats(_len), supportedgroups(_len), tick(len). In SSL_SESSION_ASN1 this *is* stored as an ASN1_OCTET_STRING though.